intelmq/lib/harmonization.py

```diff
@@ -230,6 +230,21 @@ def __parse_utc_isoformat(value):
         return value
 
     @staticmethod
+    def from_epoch_millis(tstamp, tzone='UTC'):
+        """
+        Returns ISO formatted datetime from given epoch timestamp with milliseconds.
+        It ignores the milliseconds, converts it into normal timestamp and processes it.
+        """
+        bytecount = len(str(tstamp))
+        int_tstamp = tstamp if isinstance(tstamp, int) else int(tstamp)
```

Comment:
What is that for? If you call `int()` on a int object, the same object is returned.